src/metabase/api/session.clj

```diff
@@ -313,4 +318,57 @@
         (log/error e (trs "Authentication endpoint error"))
         (throw e)))))
 
+;;; ----------------------------------------------------- Unsubscribe non-users from pulses -----------------------------------------------
+
+(def ^:private unsubscribe-throttler (throttle/make-throttler :email, :attempts-threshold 50))
```

Comment:
What should I put as the argument here? I see the login throttler uses :username https://github.com/metabase/metabase/blob/master/src/metabase/api/session.clj#L80

Reply:
Here's the [description of that arg](https://github.com/metabase/throttle/blob/master/src/throttle/core.clj#L9-L10) from the throttle source. Seems like it's to convey to the frontend which field in a form the error should be applied to. Don't think it matters for this use case so you could just make it something like `:unsubscribe`